### PR TITLE
[docker] Update CUDA version on 20.04 and add 21.04 image

### PR DESCRIPTION
**Pull request by kunaltyagi**
Let's switch the CI from 20.10 to 21.04 soon as well

### CODE REVIEW

File: .ci/azure-pipelines/env.yml [hunk 1]
```diff
@@ -46,18 +46,25 @@ jobs:
...
         UBUNTU_DISTRO: 20.10
         VTK_VERSION: 7
         # nvidia-cuda docker image has not been released for 20.10 yet
         USE_CUDA: ""
         tag: 20.10
+      Ubuntu 21.04:
+        CUDA_VERSION: 11.2.1
+        UBUNTU_DISTRO: 21.04
+        VTK_VERSION: 9
+        # nvidia-cuda docker image has not been released for 20.10 yet
```
Comment (larshg) [inline .ci/azure-pipelines/env.yml:65]:
```suggestion
        # nvidia-cuda docker image has not been released for 21.04 yet
```

**Comment by larshg** [comment]
Except minor comment error, it LGTM.

**Review by mvieth — APPROVED** [review]
Cool, I didn't know new Ubuntu releases are available on docker before the final release.
Side note: nvidia-docker cuda images seem to be only available for LTS Ubuntu releases

**Comment by kunaltyagi** [comment]
Re: CUDA: Yup, that's been a problem for a while (LTS only distribution). Hopefully, the latest CI might get nvidia support, before 21.10 rolls out 😆

**Pull request merged**